lesson_25/tommytran/template/src/pages/Home/Program.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface ProgramProps {
+  title: string;
+  description: string;
+}
+
+const Program: React.FC<ProgramProps> = ({title, description}) => {
```

Comment:
Expected to see styling for this component in an associated css file.

Reply:
new commit to fix scss issues thanks to feedback from Joseph.